src/cpp/lpnamer/model/Problem.cpp

```diff
@@ -10,3 +10,9 @@ void Problem::read_prob_mps(const std::filesystem::path& filename) {
   std::tie(mc_year, week) = McYearAndWeek(filename);
   solver_abstract_->read_prob_mps(filename);
 }
+void Problem::save_prob(const std::filesystem::path& filename) {
```

Comment:
Do we need to do `std::tie(mc_year, week) = McYearAndWeek(filename);` similarly as in the previous function ?

Reply:
Maybe, will need to check